Configuration/StandardSequences/python/Eras.py

```diff
@@ -20,6 +20,7 @@ def __init__(self):
                  'Run2_2016_trackingLowPU',
                  'Run2_2016_pA',
                  'Run2_2017',
+                 'Run2_2017Muon',
```

Comment:
I believe you should add `run2_GEMSliceTest` to `internalUseMods` below (but I can't place this comment there).